src/ast/nodes/ClassDeclaration.ts

```diff
@@ -27,6 +27,12 @@ export default class ClassDeclaration extends ClassNode {
 		if (options.systemBindings && this.id.variable.exportName) {
 			code.appendRight(this.end, ` exports('${this.id.variable.exportName}', ${this.id.variable.getName()});`);
 		}
+		if (this.id) {
+			const name = this.id.variable.getName()
+			if (name !== this.id.variable.name) {
+				code.appendRight(this.start, `let ${this.id.variable.safeName} = `)
```

Comment:
Not sure `appendRight` is the correct one. `prependLeft` doesn't work, in any case.

Reply:
Yes, `appendRight` is actually the right one. The name may feel a little counter-intuitive but it means
* insert at the current position (just like `appendLeft`, `prependRight` and `prependLeft`)
* if anything else was inserted at this position earlier, put this insertion last (this is the `append` part and certainly the right choice as we would not want anything to slip in between this insertion and the class)
* if the code *to the right*  is removed later for some other reason, remove this insertion as well (this is the `right` part and the reason why `left` would definitely be wrong here). So `right` means "attach this to the subsequent code"  while `left` means "attach this to the preceding code"